docs/fsharp/language-reference/computation-expressions.md

```diff
@@ -231,6 +255,7 @@ The following table describes methods that can be used in a workflow builder cla
 |**Method**|**Typical signature(s)**|**Description**|
 |----|----|----|
 |`Bind`|`M<'T> * ('T -> M<'U>) -> M<'U>`|Called for `let!` and `do!` in computation expressions.|
+|`MergeSources`|`M<'T> * M<'U> -> M<'T * 'U>`|Called for `and!` in computation expressions.|
```

Comment:
Please also 

1. Also add `Bind2`, `Bind3`, ... mentioned in the RFC
2. Also add `BindReturn`, `BindReturn2`, `BindReturn3`, ... mentioned in the RFC
3. Also add a link to the RFC https://github.com/fsharp/fslang-design/blob/main/FSharp-5.0/FS-1063-support-letbang-andbang-for-applicative-functors.md
4. Also add a link to  https://github.com/dsyme/fsharp-presentations/blob/master/design-notes/rethinking-applicatives.md

Reply:
If this is too much, I can add these, let us know :)